tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -27,4 +54,20 @@ void main() {
 
     expect(unpacked.aStringifiedEnum, TestEnumStringified.one);
   });
+
+  test(
+      'Given a class with a nested serverpod object when serializing and unpacking then the origin value remains unchanged.',
```

Comment:
Suggestion: Include information about the log level only being server side accessible as described in the model.

Seems like the non accessible part is what makes this test stand our from the other ones above.

Reply:
I don't think that is necessarily intentional. In any case perhaps the fundamental problem should be addressed here? Meaning the types are not accessible or we disable the entire `serverpod` type thing.